v2v-helper/nbd/nbdops.go

```diff
@@ -100,35 +107,27 @@ func StartNBDServer(ctx context.Context, server, username, password, thumbprint,
 		file,
 	)
 
-	log.Println(cmd.String())
+	// Log the command
+	cmdstring := ""
+	for _, arg := range cmd.Args {
 
-	stderr, err := cmd.StderrPipe()
-	if err != nil {
-		return NBDServer{}, err
+		if strings.Contains(arg, password) {
+			cmdstring += fmt.Sprintf("password=[REDACTED] ")
+		} else {
+			cmdstring += fmt.Sprintf("%s ", arg)
+		}
 	}
+	log.Printf("Executing %s\n", cmdstring)
 	err = cmd.Start()
 	if err != nil {
-		return NBDServer{}, err
+		return err
```

Comment:
whenever you are returning, it is helpful to add more context, I typically use 

return fmt.Errorf(" some context from this function %w", err);

so that the message you get is a concatenation of all the  places the error has travlled

Reply:
Fixed